clues.go

```diff
@@ -72,9 +72,37 @@ func AddMap[K comparable, V any](
 }
 
 // ---------------------------------------------------------------------------
-// traces
+// spans and traces
 // ---------------------------------------------------------------------------
 
+// PassTrace adds the current trace details to the provided
+// headers.  If otel is not initialized, no-ops.
+//
+// The mapCarrier is mutated by this request.  The passed
+// reference is returned mostly as a quality-of-life step
+// so that callers don't need to declare the map outside of
+// this call.
+func PassTrace[C traceMapCarrierBase](
```

Comment:
IMO, `InjectTrace` could be a more intuitive name here. It's in line with the underlying `otel.GetTextMapPropagator().Inject` call.

Reply:
sounds good!